bin/pycbc_inspiral

```diff
@@ -187,7 +189,10 @@ fft.verify_fft_options(opt,parser)
 pycbc.opt.verify_optimization_options(opt, parser)
 pycbc.weave.verify_weave_options(opt, parser)
 
-pycbc.init_logging(opt.verbose)
+if opt.info:
```

Comment:
This doesn't seem like the right solution to me. If you want different logging levels, than that logic should be in 'init_logging' itself, which could take a count of verbose levels instead of a simple on / off.